portal-web/docroot/html/taglib/ui/input_localized/page.jsp

```diff
@@ -29,19 +29,34 @@ Map<String, Map<String, String>> languagesTranslationsAriaLabelsMap = new HashMa
 					cssClass='<%= "language-value " + cssClass %>'
 					editorName="<%= editorName %>"
 					name="<%= inputEditorName %>"
+					onBlurMethod='<%= randomNamespace + "onBlurMethod" %>'
 					onChangeMethod='<%= randomNamespace + "onChangeEditor" %>'
 					onInitMethod='<%= randomNamespace + "onInitEditor" %>'
 					placeholder="<%= placeholder %>"
 					toolbarSet="<%= toolbarSet %>"
 				/>
 
 				<aui:script>
+					var edited = false;
+
+					function <%= namespace + randomNamespace %>onBlurMethod() {
+						if (edited && Liferay.FeatureFlags['LPD-11228']) {
+							edited = false;
+
+							Liferay.fire('journal:storeState', {fieldName: "<%= inputEditorName %>"});
+						}
+					}
+
 					function <%= namespace + randomNamespace %>onChangeEditor() {
-						var inputLocalized = Liferay.component('<%= namespace + HtmlUtil.escapeJS(fieldName) %>');
+						if (Liferay.FeatureFlags['LPD-11228']) {
```

Comment:
So if the FF is not enabled the `onChangeEditor() `does nothing? Could you please check this? 

I think you only need to set the `edited = true;` if the FF is enabled, and the rest should work for all the cases, isn't it? 

```
function <%= namespace + randomNamespace %>onChangeEditor() {
        if (Liferay.FeatureFlags['LPD-11228']) { edited = true; }

	var inputLocalized = Liferay.component('<%= namespace + HtmlUtil.escapeJS(fieldName) %>');

	var editor = window['<%= namespace + HtmlUtil.escapeJS(inputEditorName) %>'];

	inputLocalized.updateInputLanguage(editor.getHTML());

}

```